common/metrics/defs.go

```diff
@@ -69,6 +69,7 @@ const (
 	History
 	Matching
 	Worker
+	Server
```

Comment:
This is very unclear what Server means. We need a better name, maybe ServerExtension?

Reply:
Is this used as a tag in metrics? If so, we should try to avoid that because people are used to looking for metrics from frontend/history/matching/worker, but not from "server".

Reply:
We don't inject this tag as of now and it is not extension. Other name options is something like "Main" but it can be more ambiguous.